### PR TITLE
scrap.js: sort orgs by completed task

### PR DESCRIPTION
**Pull request by tioirawan**
this is for order orgs  by completed task #4 

Fixes https://github.com/coala/gci-leaders/issues/4

### CODE REVIEW

File: scrap.js [hunk 1]
```diff
@@ -30,6 +30,13 @@ async function fetchOrgsWithLeaders() {
...
-  const data = await fetchOrgsWithLeaders()
-  fs.writeFileSync('out/data.json', JSON.stringify(data))
+  const data = await fetchOrgsWithLeaders();
+
+  // sort data by completed_task_instance_count
+  data.sort((a, b) => 
+    b.completed_task_instance_count - a.completed_task_instance_count
+  );
+  
+
```
Comment (blazeu) [inline scrap.js]: No need to put double empty lines

File: scrap.js [hunk 2]
```diff
@@ -30,6 +30,13 @@ async function fetchOrgsWithLeaders() {
 }
 
 ;(async () => {
-  const data = await fetchOrgsWithLeaders()
-  fs.writeFileSync('out/data.json', JSON.stringify(data))
+  const data = await fetchOrgsWithLeaders();
```
Comment (blazeu) [inline scrap.js]: We don't use semicolon here ;)

File: scrap.js [hunk 3]
```diff
@@ -30,6 +30,13 @@ async function fetchOrgsWithLeaders() {
...
+
+  // sort data by completed_task_instance_count
+  data.sort((a, b) => 
+    b.completed_task_instance_count - a.completed_task_instance_count
+  );
```
Comment (blazeu) [inline scrap.js]: We don't use semicolon here ;)

File: scrap.js [hunk 4]
```diff
@@ -30,6 +30,13 @@ async function fetchOrgsWithLeaders() {
...
+  
+
+  fs.writeFileSync('out/data.json', JSON.stringify(data));
 })()
```
Comment (blazeu) [inline scrap.js]: We don't use semicolon here ;)

**Review by blazeu — CHANGES_REQUESTED** [review]

**Comment by blazeu** [comment]
Commit body is not necessary here, please remove it except for the issue reference.

Also, please start the short description with a capital letter `scrap.js: Sort ..`

**Comment by tioirawan** [comment]
Ok, sorry

File: scrap.js [hunk 5]
```diff
@@ -31,5 +31,8 @@ async function fetchOrgsWithLeaders() {
 
 ;(async () => {
   const data = await fetchOrgsWithLeaders()
+  data.sort((a, b) =>
```
Comment (blazeu) [inline scrap.js]: Why did you remove the comment?

**Comment by blazeu** [comment]
Squash it into a single commit, please.

**Comment by tioirawan** [comment]
how? sorry

File: scrap.js [hunk 6]
```diff
@@ -31,5 +31,9 @@ async function fetchOrgsWithLeaders() {
 
 ;(async () => {
   const data = await fetchOrgsWithLeaders()
+  // sort data by completed_task_instance_count
```
Comment (blazeu) [inline scrap.js]: I was meant to say to make the double empty lines to a single one, sorry.

Please do keep the previous empty lines, except that double ones, make it into a single line.

**Comment by blazeu** [comment]
@Indmind http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

if you're still confused, ask away in our chatroom :)

**Comment by Indmind** [comment]
screenshot from 2017-12-05 10-25-12

**Comment by blazeu** [comment]
I think you didn't put the correct command for each commit.

Please ask and get help in our chatroom, preferably on zulip. Although I wont be there (not in my pc rn), there's another mentor that can help

**Comment by tioirawan** [comment]
Done :)

**Comment by blazeu** [comment]
Nice! 🎉  I'll approve your task in GCI,
but merging comes later, I'm outside.

Please submit your task in GCI dashboard.

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/10/commits/4c4a028bdb7e2de2385ad4a9c30bea99db7fff3e

**Comment by blazeu** [comment]
Hey, I just noticed that your commit body is a mess.
Please tidy it just like before, but keep it in 1 commit.

**Comment by blazeu** [comment]
unack https://github.com/coala/gci-leaders/pull/10/commits/4c4a028bdb7e2de2385ad4a9c30bea99db7fff3e

**Comment by tioirawan** [comment]
sorry, I didn't notice that

**Comment by blazeu** [comment]
I just setup a deploy preview.
Please do `git commit --amend` and repush so we can see the preview :grinning:

**Review by prnvdixit — COMMENTED** [review]
Also, keep in mind to not push changes from your master branch - create a new branch instead. Might lead to some conflicts. Refer http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-3-creating-a-fork-and-testing-your-changes. :smile:

**Comment by prnvdixit** [comment]
Also, it's not labelled "bug" - So, use "Closes" instead of "Fixes" in commit. 👍

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/10/commits/b03b46d649c0dce92fbf4bbe402fbeb19e04855f

**Comment by blazeu** [comment]
Now, you just need to do rebase, so that it can be merged. See https://coala.io/rebase
